Coalition.rte/Devices/Weapons/Pistols/MuzzleSmoke.lua

```diff
@@ -0,0 +1,22 @@
+function Create(self)
+	local Effect
+	local Offset = self.Vel*(20*TimerMan.DeltaTimeSecs)/1.3	-- the effect will be created the next frame so move it one frame backwards towards the barrel
```

Comment:
Blah blah duplicate script. Funnily the math here is slightly different it seems - n/1.3 instead of n/1.2